plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -4886,3 +4886,100 @@ func TestInterPodAffinityAnnotationsWithMultipleNodes(t *testing.T) {
 		}
 	}
 }
+
+func benchmarkPodMatchesNodeLabels(pods []*v1.Pod, nodes []*v1.Node, b *testing.B) {
```

Comment:
So I'm ok with micro-benchmarks here so long as they measure against some level of regression, or are recorded.  Otherwise there are separate performance tests that exist for the scheduler.